pkg/network/node_identity.go

```diff
@@ -89,8 +89,24 @@ func renderNetworkNodeIdentity(conf *operv1.NetworkSpec, bootstrapResult *bootst
 	// HyperShift specific
 	if hcpCfg := hypershift.NewHyperShiftConfig(); hcpCfg.Enabled {
 		webhookCAClient = client.ClientFor(names.ManagementClusterName)
-		webhookCALookup = types.NamespacedName{Name: "openshift-service-ca.crt", Namespace: hcpCfg.Namespace}
-		caKey = "service-ca.crt"
+		CAConfigMap, ok := os.LookupEnv("CA_CONFIG_MAP")
```

Comment:
This should rely on the `hypershift.go` implementation instead of duplicating the logic.

Reply:
Updated